package.json

```diff
@@ -7,8 +7,8 @@
   "description": "Post preview service for Hashnode",
   "type": "module",
   "engines": {
-    "node": ">=20",
-    "npm": ">=10"
+    "node": ">=22",
+    "pnpm": ">=10"
   },
```

Comment:
```suggestion
  },
  "packageManager": "pnpm@10.11.0+sha512.6540583f41cc5f628eb3d9773ecee802f4f9ef9923cc45b69890fb47991d4b092964694ec3a4f738a420c918a333062c8b925d312f42e4f0c263eb603551f977",
```

Reply:
Thanks for catching that, @raisedadead. LGTM 👍